galaxy/settings/production.py

```diff
@@ -210,6 +215,19 @@ def _set_logging():
 
 METRICS_ENABLED = True
 
+
+# =========================================================
+# InfluxDB Settings
+# =========================================================
+INFLUX_DB_HOST = os.environ.get('GALAXY_INFLUX_DB_HOST', 'influxdb')
+INFLUX_DB_PORT = os.environ.get('GALAXY_INFLUX_DB_PORT', '8086')
+INFLUX_DB_USERNAME = os.environ.get('GALAXY_INFLUX_DB_USERNAME', 'admin')
+INFLUX_DB_PASSWORD = os.environ.get('GALAXY_INFLUX_DB_PASSWORD', '')
+INFLUX_DB_DATABASE_NAME = os.environ.get(
```

Comment:
Since we're using multiple influxdb databases, parameter name should reflect database purpose (e.g. GALAXY_INFLUX_UI_EVENTS_DB`